app/points/calculate-weighting.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (weightingPercentage) {
+  var weighting
+  if (weightingPercentage >= 100) {
+    weighting = 0
+  } else if (weightingPercentage <= 0) {
+    weighting = 1
+  } else {
+    weighting = (100 - weightingPercentage) / 100
+  }
+  return weighting
+}
```

Comment:
Cheers for the input. Can we agree on single exit point going forward then? (as long as it doesn't make something massively unreadable obviously) @dconey646 @scoch